### PR TITLE
Add ai-humanizer extension

### PR DESCRIPTION
**Pull request by frolvanya**
## Description
This extension allows you to easily humanize text using an AI-powered service. Simply paste your original text, click the "Humanize" button, and the humanized result will be displayed. You can then copy the humanized text to the clipboard with a single shortcut.

## Screencast
ai-humanizer-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/ai-humanizer/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <Form
+      isLoading={loading}
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm title="Humanize" onSubmit={handleSubmit} />
+          <Action.CopyToClipboard content={humanizedText} />
+        </ActionPanel>
+      }
+    >
+      <Form.TextArea autoFocus={true} id="originalText" title="Original text" />
```
Comment (pernielsentikaer) [inline extensions/ai-humanizer/src/index.tsx:47]:
```suggestion
      <Form.TextArea autoFocus={true} id="originalText" title="Original Text" />
```

File: extensions/ai-humanizer/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <Form.Separator />
+      <Form.TextArea id="humanizedText" title="Humanized text" value={humanizedText} />
```
Comment (pernielsentikaer) [inline extensions/ai-humanizer/src/index.tsx:49]:
```suggestion
      <Form.TextArea id="humanizedText" title="Humanized Text" value={humanizedText} />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Could you make the [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) with our tool instead



I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by frolvanya** [comment]
I've updated a screenshot and applied suggested changes
https://github.com/raycast/extensions/blob/0646be56a268442e1f52a5c46333acf8449812a4/extensions/ai-humanizer/src/index.tsx#L46
https://github.com/raycast/extensions/blob/0646be56a268442e1f52a5c46333acf8449812a4/extensions/ai-humanizer/src/index.tsx#L48

**Review by pernielsentikaer — APPROVED** [review]
✅ Approved

**Comment by frolvanya** [comment]
Thanks! I just linked my account

**Comment by pernielsentikaer** [comment]
Then it's automatically added 🙂

**Pull request merged**